docs/commandline_interface.rst

```diff
@@ -0,0 +1,54 @@
+Command-line interface
+======================
+
+pytube also ships with a tiny cli interface for downloading and probing
```

Comment:
Pytube should be capitalized since it's the start of a sentence, CLI should also be in caps